internal/config/config.go

```diff
@@ -4,23 +4,24 @@ import (
 	"log"
 	"os"
 
+	"github.com/kauefraga/anubis/internal/models"
 	"github.com/pelletier/go-toml/v2"
 )
 
 type Config struct {
 	Version   int
 	Port      uint16
 	Algorithm string
-	Servers   []string
+	Servers   []*models.Servers
```

Comment:
How this structure looks like in the TOML file?

Reply:
like this 
[[servers]]
url = "localhost:8080"

[[servers]]
url = "localhost:8081"